README.md

```diff
@@ -1,3 +1,19 @@
-## Organizing a Collaborator Summit
+# Node.js Collaborator Summit
+
+The Node.js Foundation hosts regular summits for active contributors to the node.js project, this repository is being used to organize them.
+
+## Cost
+Free!
+
+Although the summits may be coordinated with other events/confrences, you do not need to pay/attend those events to participate in a summit.
+
+## Upcoming Events
+- [Aug 17-18 2016, Amsterdam, Netherlands](https://github.com/nodejs/summit/issues/16)
+- Dec 01-02 2016, Austin, Texas, USA
+
+## Past Events
+- [Aug 06-07 2015, San Francisco, California](https://github.com/nodejs/summit/issues/1)
+
+## Code of Conduct
+See https://github.com/nodejs/summit/issues/2
```

Comment:
I would simply link to http://events.linuxfoundation.org/events/node-interactive-europe/attend/code-of-conduct